third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -665,7 +674,7 @@ struct ScaledDotOpMFMAConversionHelper : DotOpMFMAConversionHelper {
       setPrioOp->moveAfter(firstMfma.getDefiningOp());
 
     const size_t mmaCount =
-        numRepB * numRepM * numRepN * numRepK * kWidth / kBase;
+        numRepB * numRepM * numRepN * numRepK * aKWidth / aKBase;
     packAndReplaceResult(op, fc, maybeMfmaIntrinsic, dstElemTy, elemTyA,
```

Comment:
Not sure if we need to change the `elemTyA` here or not. Mixed input types are supported in this PR. Only the element type of A might not be representive enough. But looks like it's just for hints or telemetry...

Reply:
This part is some prior experiments for helping normal dot instruction scheduling on mi300. for mi350 it's unlikely to be effective so fine to ignore for now.